### ISSUE TITLE
tailscale: `ip6tables` error using linux-6.6.58_1

### ISSUE THREAD

**Issue by invakid404** [void-linux/void-packages#52886]
### Is this a new report?

Yes

### System Info

Void 6.6.58_1 x86_64 AuthenticAMD uptodate rrrrmmnFFFFF

### Package(s) Affected

tailscale-1.76.3_1 linux-6.6_1

### Does a report exist for this bug with the project's home (upstream) and/or another distro?

https://github.com/tailscale/tailscale/issues/13863

### Expected behaviour

All Tailscale features should work with Linux 6.6.58_1.

### Actual behaviour

Certain Tailscale features (subnet routing, MagicDNS) don't work when running Linux 6.6.58_1.

### Steps to reproduce

1. Ensure you are running `linux-6.6.58_1`
2. Install and configure `tailscale`.
3. Run `tailscale status` and observe the following warning:

```
# Health check:
#     - adding [-i tailscale0 -j MARK --set-mark 0x40000/0xff0000] in v6/filter/ts-forward: running [/usr/bin/ip6tables -t filter -A ts-forward -i tailscale0 -j MARK --set-mark 0x40000/0xff0000 --wait]: exit status 2: Warning: Extension MARK revision 0 not supported, missing kernel module?
ip6tables v1.8.10 (legacy): MARK: bad value for option "--set-mark", or out of range (0-4294967295).
```

**Comment by invakid404** [comment]
The Tailscale issue linked above contains all the necessary context, but to summarize:

- The issue is caused by https://github.com/torvalds/linux/commit/0bfcb7b71e735560077a42847f69597ec7dcc326
- Users have confirmed that applying the following [patch](https://lore.kernel.org/all/20241019-xtables-typos-v3-1-66dd2eaacf2f@0upti.me/) resolves the issue
- Linux 6.6.59 is supposed to contain the fix already, so upgrading to it should be sufficient to resolve this issue

**Comment by classabbyamp** [comment]
should be fixed by bd52cd2a8e6bc58953a8a5ddb7eb6f58640c8f5c, can you confirm?

**Comment by invakid404** [comment]
> should be fixed by [bd52cd2](https://github.com/void-linux/void-packages/commit/bd52cd2a8e6bc58953a8a5ddb7eb6f58640c8f5c), can you confirm?

Yep, it works with 6.6.59. Thanks!
